src/preview/index.js

```diff
@@ -39,10 +42,26 @@ style.innerHTML = `
 document.head.appendChild(style);
 previewCallback();
 
+const StyledWrapper = styled(Box)`
+  cursor: ${(props) =>
```

Comment:
Is this required? I'm seeing curser set elsewhere too

Reply:
It's required just because we need the cursor on hover to show up to infer that the panel is resizeable but then we need it on the whole container, too, so that while you're resizing if your cursor goes outside of the resize area it still shows the correct cursor